loops-and-arrays.js

```diff
@@ -5,6 +5,9 @@
 // ==========================================
 
 const names = ["Henk", "Piet", "Fred", "Joop"];
+for (let i = 0; i <= names.length-1; i++){
+    console.log(names[i]+"je")
```

Comment:
Ik heb dit op dezelfde manier gedaan, ik merk dat het bij de uitwerkingen vaker verwerkt is in een aparte variabele, voordat het wordt gelogd. Ik weet niet of het incorrect is.